### PR TITLE
removes exjs dependency

### PR DESCRIPTION
**Pull request by mejackreed**
Part of #41 

Though, I'm not sure how much this does in reducing actual bundle size.

### CODE REVIEW

**Comment by edsilv** [comment]
Hey, please see the failing `getRangeByPath` test I added.
I think it's related to this: https://github.com/IIIF-Commons/manifesto/pull/42/files#diff-01f13f6d1a0f04758601d8128e56b2efL1429

File: src/Manifest.ts [hunk 1]
```diff
@@ -176,8 +176,8 @@ namespace Manifesto {
                 if (topRange.id){
                     this._allRanges.push(topRange); // it might be a placeholder root range
                 }
-                const subRanges: IRange[] = topRange.getRanges();        
-                this._allRanges = this._allRanges.concat(subRanges.en().traverseUnique(range => range.getRanges()).toArray());
+                const subRanges: IRange[] = topRange.getRanges();
+                this._allRanges = this._allRanges.concat(subRanges);
```
Comment (stephenwf) [inline src/Manifest.ts]: Could reduce it down using something like:
```js
const reducer = (acc, next) => {
  acc.push(next);
  const nextRanges = next.getRanges();
  if (nextRanges.length) {
     return nextRanges.reduce(reducer, acc);
  }
  return acc;
}
```

and use it like:
```js
const flatRanges = topRange.getRanges.reduce(reducer, [topRange]);
```

If it needs to be unique, could make it a `Set` or a `Map` mapped by the id field. (if this is the failing test that you mentioned)

File: src/LabelValuePair.ts [hunk 2]
```diff
@@ -27,7 +27,8 @@ namespace Manifesto {
 
         public setLabel(value: string): void {
             if (this.label && this.label.length) {
-                var t: Manifesto.Language = this.label.en().where(x => x.locale === this.defaultLocale || x.locale === Manifesto.Utils.getInexactLocale(this.defaultLocale)).first();
+                console.log(this.label);
```
Comment (stephenwf) [inline src/LabelValuePair.ts]: A couple of console logs made its way in 👀

Comment (mejackreed) [reply src/LabelValuePair.ts]: Thanks for catching that.

File: src/Manifest.ts (hunk 1 above)
Comment (mejackreed) [reply src/Manifest.ts]: Thanks for the comments here. I've used your approach @stephenwf  and pushed that up in 
4576dc8.

**Comment by mejackreed** [comment]
Thanks for the great review (and the additional tests added). Is there a way you would like me to resolve the `dist` conflicts?

Also I noticed that the uniqueness needed in the added test wasn't caught when just using an `Array`. @edsilv is there a manifest/scenario where I could write a failing test for that?

**Comment by edsilv** [comment]
I've altered the biocrats.js test to check the number of returned ranges. The root range was being included when it didn't have an `id`. Root ranges without `ids` are "placeholders" for generating trees I believe.

I've also removed the dist directory, but this doesn't seem to have fixed the conflicts?

**Comment by mejackreed** [comment]
I've force pushed a rebase that should handle the `dist` thrashing. I'm going to submit a separate PR that should help with the release process for this package, so that built versions do not need to be submitted in PR's or committed.

**Comment by edsilv** [comment]
Cool. Well, the UV seems happy now. Think it's mergeable?

**Pull request merged**